genestack_client/genestack_exceptions.py

```diff
@@ -26,6 +26,12 @@ def __init__(self, message, path, post_data, debug=False, stack_trace=None):
         :param stack_trace: server stack trace
         :type stack_trace: str
         """
+        def to_str(val):
+            if isinstance(val, unicode):
+                return val.encode('utf-8', 'ignore')
+            return val
+        message = to_str(message)
+
         GenestackException.__init__(self, message, path, post_data, debug, stack_trace)
         self.message = message
```

Comment:
In superclass this stored in `args` attribute.  Passing arguments to superclass required for pickling exception if it is used in threads. http://stackoverflow.com/a/28335286